src/Microsoft.HttpRepl/Telemetry/TelemetryExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.HttpRepl.Telemetry.Events;
+
+namespace Microsoft.HttpRepl.Telemetry
+{
+    internal static class TelemetryExtensions
+    {
+        public static void TrackEvent(this ITelemetry telemetry, TelemetryEventBase telemetryEvent)
```

Comment:
I was wondering why this isn't just a method on ITelemetry, is it because it's defined in the Telemetry assembly but TelemetryEventBase is defined in the HttpRepl assembly?  (If so, I'm fine with this, just wanted to clarify the need to implement via extension method)

Reply:
Yep, that's right. Keeping the HttpRepl-specific stuff (like TelemetryEventBase) separate from the shared-by-source pieces that are in the Telemetry assembly.